### DISCUSSION TITLE
Need explanation of feature types

### DISCUSSION THREAD

**Discussion by farconada** [awslabs/gluonts#1499] [Q&A]
Hello

Can anyone explain (with examples, please) how to use the multiple feature types?

For example: 
- I want to add a feature "days since last outlier"
- another feature: is summer
- moving average of 12 steps
- value of another serie in period t-1
thanks

**Comment by pratikgehlott** [comment]
hello @farconada can you be more specific about what you need to know?

Thank you.

**Reply by farconada** [reply]
for example,
I have a time serie (one with my personal expenses for years with a month frequency). I have annotated months with are outliers for example: I broke my car on may 2019 (this is an outlier cause that month I had to pay a lot to repair it); another one, I traveled for for holidays cause previous months my economy was so regular; etc etc
I want to have a feature "months since last outlier" cause I think "outliers" happen with some kind of pattern. 

How should I code that feature with gluonts? could you provide an example?

Thanks

**Reply by pratikgehlott** [reply]
Ahh, you need what frequency should you put soo that your data can fit into gluonts format?
